src/asim/configs/common_airport/write_trip_matrices_annotate_trips_preprocessor.csv

```diff
@@ -211,7 +211,7 @@ Description,Target,Expression
 #,_timeDrive,"_timeDrive + (odt_skims['HOV2_M_TIME'] + _TAXI_WAIT_TIME) * np.where((trip_mode == 'TAXI'),1,0)"
 #,_timeDrive,"_timeDrive + (odt_skims['HOV2_M_TIME'] + _SINGLE_TNC_WAIT_TIME) * np.where((trip_mode == 'TNC_SINGLE'),1,0)"
 ,_timeDrive,"_timeDrive + odt_skims['HOV3_M_TIME'] * np.where(((trip_mode == 'SHARED3') & vot2),1,0)"
-#,_timeDrive,"_timeDrive + (odt_skims['HOV3_M_TIME'] + _SHARED_TNC_WAIT_TIME) * np.where((trip_mode == 'TNC_SHARED'),1,0) * _TNC_SHARED_IVT_FACTOR"
+#,_timeDrive,"_timeDrive + (odt_skims['HOV3_M_TIME'] + _SHARED_TNC_WAIT_TIME) * TNC_shared_IVTFactor * np.where((trip_mode == 'TNC_SHARED'),1,0) * _TNC_SHARED_IVT_FACTOR"
```

Comment:
Please check if _TNC_SHARED_IVT_FACTOR is still needed.

Reply:
_TNC_SHARED_IVT_FACTOR no longer exists, this line is commented out

Reply:
No TNC_SHARED mode in airport model

Reply:
Removed these commented out lines to avoid confusion